docs/content/manual/pre-release/resiliency/simulated-slow-disk.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "[#2206](https://github.com/longhorn/longhorn/issues/2206) Fix the spinning disk on Longhorn"
+---
+
+This case requires the creation of a slow virtual disk with `dmsetup`.
+1. Make a slow disk:
+    - Make a disk image file: `truncate -s 10g slow.img`
+    - Create a loopback device: `losetup --show -P -f slow.img`
+    - Get the block size of the loopback device: `blockdev --getsize /dev/loopX`
+    - Create slow device: `echo "0 <blocksize> delay /dev/loopX 0 500" | dmsetup create dm-slow`
+    - Format slow device: `mkfs.ext4 /dev/mapper/dm-slow`
+    - Mount slow device: `mount /dev/mapper/dm-slow /mnt`
+2. Build longhorn-engine and run it on the slow disk.
+    - `make` or use a tag from docker hub in next step.
+    - `docker run --privileged -v /dev:/host/dev -v /proc:/host/proc -v /mnt:/volume longhornio/longhorn-engine:<tag> launch-simple-longhorn slow-vol 10g tgt-blockdev`
+3. Perform intense I/O and verify that it doesn't fail.
```

Comment:
I'm not sure but do we want to add a test case here with holes in the sparse file? In the past we've seen it impacts the performance.

Reply:
I'm not sure.  The sparse file used to store the replica should have holes in it while the fio command is running.   Do you have a description of a test with the sparse file having holes impacting performance?  I'm not sure how to recreate it with this direct I/O on the block device file example.

Reply:
I used below fio job to create sparse file having holes, I assume that `--rw=write:4k` will help.
```
[global]
bs=4k
iodepth=128
direct=1
sync=0
end_fsync=1
ioengine=libaio
randseed=1
randrepeat=0
# buffer_pattern="X"
group_reporting

# 4k bs
[job1]
bs=4k
rw=write:4k
filename=/dev/longhorn/rebuild-4k-old
name=data4k-holes4k-test
size=5G
```

Reply:
I added a test with --rw=write:4k